elixir/apps/web/lib/web/components/form_components.ex

```diff
@@ -159,6 +152,23 @@ defmodule Web.FormComponents do
     """
   end
 
+  attr :id, :string, default: nil
+  attr :name, :string, default: nil
+  attr :value, :string, default: nil
+  attr :checked, :boolean, default: false
+
+  def checkbox(assigns) do
+    ~H"""
+    <input
+      type="checkbox"
+      name={@name}
+      value={@value}
+      checked={@checked}
```

Comment:
You don't have to define all params HTML support, if you don't plan to do anything with the values you can use a global attribute:
```
attr :rest, :global
``
and then you can just use it to pass any attrs that are not defined
```suggestion
      {@rest}
```
see https://hexdocs.pm/phoenix_live_view/Phoenix.Component.html#module-global-attributes

Reply:
Ah, ok, good to know!  I'll get this updated.

Reply:
Note for posterity:

As discussed offline, the global attributes in phoenix by default only include the [HTML global attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes).  That means the `name`, `value`, and `checked` attributes aren't included.  There is a way to extend what is [included in the phoenix global attributes](https://hexdocs.pm/phoenix_live_view/Phoenix.Component.html#module-included-globals), but for the time being, since the generated components in this same file explicitly define the above mentioned attributes, I'll also define them for the `checkbox` component, just for consistency.